<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisDataWidget</class>
 <widget class="QWidget" name="AxisDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="label_source">
       <property name="minimumSize">
        <size>
         <width>185</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_selectSource">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/tree-settings.png</normaloff>:/tree-settings.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clearSource">
       <property name="toolTip">
        <string>Clear selection</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/dialog-close.png</normaloff>:/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>min:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spin_unitMin"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>max:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spin_unitMax"/>
     </item>
     <item>
      <widget class="QLabel" name="label_unit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combo_resolution"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
